pep-0563.rst

```diff
@@ -259,14 +259,15 @@ valid.  They can use local names or the fully qualified name.  Example::
             def method(self) -> C.D.field2:  # this is OK
                 ...
 
-            def method(self) -> D.field2:  # this is OK
-                ...
+            def method(self) -> D.field2:  # this FAILS, class D is local to C 
+                ...                        # and is therefore only available 
+                                           # as C.D.
 
             def method(self) -> field2:  # this is OK
                 ...
 
             def method(self) -> field:  # this FAILS, class D doesn't
-                ...                     # see C's attributes,  This was
+                ...                     # see C's attributes.  This was
                                         # already true before this PEP.
```

Comment:
Why mention this here and not in the previous example? And why not show the correct way (C.field) like you did there?